docs/content/guides/getting-started/introduction.md

```diff
@@ -39,15 +39,15 @@ Thousands of business apps depend on Handsontable for entering, editing, validat
 ::: only-for javascript
 To jump straight into the sample code, open the demo app at CodeSandbox: 
 
-- [JavaScript demo](https://codesandbox.io/s/handsontable-javascript-data-grid-hello-world-app-forked-kfcm0r)
-- [React demo](https://codesandbox.io/s/handsontable-react-data-grid-hello-world-app-forked-u2168i)
-- [Angular demo](https://codesandbox.io/s/handsontable-angular-data-grid-hello-world-app-forked-cju8bf)
-- [Vue 2 demo](https://codesandbox.io/s/handsontable-vue-data-grid-hello-world-app-forked-6sunw5)
-- [TypeScript demo](https://codesandbox.io/s/handsontable-typescript-data-grid-hello-world-app-forked-7t4jdz)
+- [JavaScript demo](https://codesandbox.io/s/handsontable-javascript-data-grid-hello-world-app-forked-8tct2l)
+- [React demo](https://codesandbox.io/s/handsontable-react-data-grid-hello-world-app-forked-56qvdt)
+- [Angular demo](https://codesandbox.io/s/handsontable-angular-data-grid-hello-world-app-forked-910xle)
+- [Vue 2 demo](https://codesandbox.io/s/handsontable-vue-data-grid-hello-world-app-forked-k6h5v9)
+- [TypeScript demo](https://codesandbox.io/s/handsontable-typescript-data-grid-hello-world-app-forked-z8dz1r)
```

Comment:
@aninde the above CodeSandboxes will not work until the `12.3.0` package is published on NPM. 

Please make sure to run a test case (after each release) to see if CodeSandbox links work OK.